qa/tasks/cephfs/test_nfs.py

```diff
@@ -1212,3 +1217,64 @@ def test_cephfs_export_update_at_non_dir_path(self):
         finally:
             self.ctx.cluster.run(args=['rm', '-rf', f'{mnt_pt}/*'])
             self._delete_cluster_with_fs(self.fs_name, mnt_pt, preserve_mode)
+
+    def test_nfs_export_creation_without_cmount_path(self):
+        """
+        Test that ensure cmount_path is present in FSAL block
+        """
+        pseudo_path = '/test_without_cmount'
+        self._create_export(export_id='1234',
+                            extra_cmd=['--pseudo-path', pseudo_path])
+        nfs_output = self._get_export(pseudo_path)
+        self.assertIn('cmount_path', nfs_output['fsal'])
+
+        self._delete_export(pseudo_path)
+
+    def test_nfs_create_abd_del_exports_with_same_and_diff_user_id(self):
```

Comment:
typo, also try to make this shorter

Reply:
does it really matter if not tht shorter? I just tried to cover both create & delete in one

Reply:
you can break down test cases for brevity. `test_nfs_create_del_exports_with_same_userid`, `test_nfs_create_del_exports_with_diff_userid`

Reply:
I simply changed to `test_nfs_exports_with_same_and_diff_user_id` and fixed typo

Reply:
I'd rather prefer to clear ids in same test case. PTAL at changes I did

Reply:
I don't think this is a feasible idea to mix two test cases together IMO (it also makes debugging failures hard later on) but okay, I'll approve it.